stl/inc/algorithm

```diff
@@ -465,7 +465,8 @@ constexpr bool _Vector_alg_in_unique_is_safe = _Vector_alg_in_search_is_safe<_It
 // Can we use this output iterator for remove_copy and unique_copy?
 template <class _Out, class _In>
 constexpr bool _Output_iterator_for_vector_alg_is_safe() {
-    if constexpr (_Iterator_is_contiguous<_Out>) {
+    if constexpr (is_pointer_v<_Out>) { // P3349R1: Non-pointer contiguous iterators prevent vectorization (unknown

```

Comment:
I don't understand this part.. But also we don't usually cite papers, except in yvals_core.h

Reply:
This also has some relation to LWG-4262. There are comment of that one being NAD, 

If this is resolved as NAD, then we should probably add the source range size.

Reply:
> It looks like this part has changed in the latest working draft:

Interesting! I want to know which paper(s) cause this change. LWG-4262 appears to be no longer needed.

Reply:
It was WG21-P3179 that changed this part.

Reply:
Thanks, I'll adjust the comment; I agree that we generally don't cite papers (which becomes increasingly confusing as the Standard is edited over the years). I think that the actual impact of limiting vectorization to pointers after unwrapping, instead of arbitrary contiguous iterators that aren't aware of unwrapping, is limited - 99% of use cases will be with pointers, arrays, and STL iterators that participate in unwrapping (`array`, `string`, `string_view`, `vector` iterators).